pkgs/development/python-modules/myfitnesspal/default.nix

```diff
@@ -1,11 +1,13 @@
-{ lib, fetchPypi, buildPythonPackage
+{ lib, fetchPypi, buildPythonPackage, isPy3k
 , blessed, keyring, keyrings-alt, lxml, measurement, python-dateutil, requests, six
 , mock, nose }:
 
 buildPythonPackage rec {
   pname = "myfitnesspal";
   version = "1.13.4";
 
+  disabled = !isPy3k;
```

Comment:
this isn't really needed, as measurement will transitively make this python3 only

Reply:
Removed